content/docs/self-hosted-runners.md

```diff
@@ -207,25 +243,52 @@ Click below to see credentials needed for supported cloud service providers.
 
 </details>
 
-## Using on-premise machines as self-hosted runners
+<details>
+
+#### GCP
+
+Either one of:
+
+- `GOOGLE_APPLICATION_CREDENTIALS_DATA`: the **contents** of a service account
+  JSON file, or
+- `GOOGLE_APPLICATION_CREDENTIALS`: the **path** to the JSON file.
+
+The former is more convenient for CI/CD scenarios, where secrets are (usually)
+provisioned through environment variables instead of files.
+
+</details>
+
+<details>
+
+#### K8s
```

Comment:
```suggestion
#### Kuberentes
```

Reply:
```diff
- kuberentes
+ kubernetes
```

Reply:
🤦🏼 ✖️ 100; code/text review and meetings aren't meant to be parallelized.

Reply:
mean what idk you

![](https://imgs.xkcd.com/comics/future.png)

Reply:
My keyboard is full of race conditions. It must be a hardware issue... with my brain. 😄